pegasus/sites.v3/csedweek.org/styles/040-styling.css

```diff
@@ -31,6 +31,8 @@ h2 {
   font-size: 22px;
   font-family: 'Gotham 5r', sans-serif;
   line-height: 1.2em;
+  margin-top: 45px;
```

Comment:
these two lines actually make it match code.org better, the rest of the file is just white space